content/W2017/W17-event-Techtalk-with-Monica-Bechwith.md

```diff
@@ -0,0 +1,16 @@
+Title: Techtalk with Monica Beckwith
+Date: 2017-03-14
+Category: Events
+Tags: social
+Slug: tech-talk-w17
+Author: Fatima Taj
+Summary: Come join us at a tech talk with Monica Beckwith!
+
+Come learn about JIT, Runtime Optimizations, and HotSpot garbage collection with Monica Beckwith!
+
+## Event Details ##
+
++ **Who:** Monica Beckwith
```

Comment:
This should detail who is permitted to attend the event, in this case everyone 😊